header.php

```diff
@@ -19,6 +19,8 @@
 
 <body <?php body_class(); ?>>
 
+<?php wp_body_open(); ?>
```

Comment:
@kishanjasani 

> Note that if your theme is going to be submitted to the theme repository, then you won’t be able to use the wp_ prefix, as it will get flagged by the Theme Check. An alternative is to call do_action directly where the wp_body_open() function is placed in the first example, like this:

Please use this code instead https://make.wordpress.org/core/2019/04/24/miscellaneous-developer-updates-in-5-2/#highlighter_807826

Reply:
Did as suggested in the documentation. :+1: